Add files via upload

andreimuth opened the pull request:
I need some help at 3.6 and 3.8...

andreimuth committed:
Add files via upload

I need some help at 3.6 and 3.8...

Alex-Cosma commented:
You will have to be a bit more specific than this. What exactly do you need help with?

andreimuth committed:
Add files via upload

I succeeded to solve 3.8. At 3.6 I tried to "divide" a float number in two integers (for example 4.53 -> 4 and 53), but I encounter errors at the fractional part...

andreimuth committed:
Add files via upload

Problem 3.8

andreimuth committed:
Add files via upload

Problem 3.6

Alex-Cosma commented:
I don't have the time to try it out myself but the code looks good.
Does the approach work? All good?

andreimuth committed:
Add files via upload

3.3

andreimuth committed:
Add files via upload

3.3 *

Alex-Cosma commented:
Good
